GameData/002_CommunityPartsTitles/Localization/CPT_NearFutureSpacecraft_LaunchVehicles_loc.cfg

```diff
@@ -103,35 +103,35 @@ Localization
-		#LOC_NFLaunchVehicles_nflv-adapter-75-5-2_title           = Kerbodyne S4-S5-117 Adapter Tank (E-Series)  // EA-S10 Fuel Tank Adapter
-		#LOC_NFLaunchVehicles_nflv-adapter-75-5-1_title           = Kerbodyne S4-S5-234 Adapter Tank (E-Series)  // EA-S20 Fuel Tank Adapter
-		#LOC_NFLaunchVehicles_nflv-fueltank-round-75-1_title      = Kerbodyne S5 216 Rounded Nosecone            // EA-C-64 Rounded Nosecone
-		#LOC_NFLaunchVehicles_nflv-fueltank-nosecone-75-1_title   = Kerbodyne S5 432 Fuelled Nosecone            // EA-C-128 Fuelled Nosecone
-		#LOC_NFLaunchVehicles_nflv-fueltank-75-4_title            = Kerbodyne S5-0096 Fuel Tank (E-Series)       // EA-F96 Fuel Tank
-		#LOC_NFLaunchVehicles_nflv-fueltank-75-3_title            = Kerbodyne S5-0192 Fuel Tank (E-Series)       // EA-F192 Fuel Tank
-		#LOC_NFLaunchVehicles_nflv-fueltank-75-2_title            = Kerbodyne S5-0384 Fuel Tank (E-Series)       // EA-F384 Fuel Tank
-		#LOC_NFLaunchVehicles_nflv-fueltank-75-1_title            = Kerbodyne S5-0768 Fuel Tank (E-Series)       // EA-F768 Fuel Tank
-		#LOC_NFLaunchVehicles_nflv-fueltank-75-5_title            = Kerbodyne S5-1536 Fuel Tank (E-Series)       // EA-F1536 Fuel Tank
+		#LOC_NFLaunchVehicles_nflv-decoupler-75-1_title           = TD-75 Decoupler (N-Series)               // TD-750 Stack Decoupler
+		#LOC_NFLaunchVehicles_nflv-separator-75-1_title           = TS-75 Separator (N-Series)               // TS-750 Stack Separator
+		#LOC_NFLaunchVehicles_nflv-fairing-75-1_title             = AE-FF5 Payload Fairing (7.5m) (N-Series) // AE-FF5 Payload Fairing (7.5m)
+
+		#LOC_NFLaunchVehicles_nflv-skeletal-adapter-75-5-1_title  = FV-50-75-k12 "SKL" Adapter Tank (N-Series)             // EA-AD-SKL Adapter
+		#LOC_NFLaunchVehicles_nflv-adapter-75-5-2_title           = FV-50-75-k12 Adapter Tank (N-Series)  // EA-S10 Fuel Tank Adapter
+		#LOC_NFLaunchVehicles_nflv-adapter-75-5-1_title           = FV-50-75-k23 Adapter Tank (N-Series)  // EA-S20 Fuel Tank Adapter
+		#LOC_NFLaunchVehicles_nflv-fueltank-round-75-1_title      = FS-75-k022 Rounded Nosecone (N-Series)           // EA-C-64 Rounded Nosecone
+		#LOC_NFLaunchVehicles_nflv-fueltank-nosecone-75-1_title   = FS-75-k043 Fuelled Nosecone (N-Series)           // EA-C-128 Fuelled Nosecone
+		#LOC_NFLaunchVehicles_nflv-fueltank-75-4_title            = FL-75-k014 Fuel Tank (N-Series)       // EA-F96 Fuel Tank
+		#LOC_NFLaunchVehicles_nflv-fueltank-75-3_title            = FL-75-k029 Fuel Tank (N-Series)       // EA-F192 Fuel Tank
+		#LOC_NFLaunchVehicles_nflv-fueltank-75-2_title            = FL-75-k058 Fuel Tank (N-Series)       // EA-F384 Fuel Tank
+		#LOC_NFLaunchVehicles_nflv-fueltank-75-1_title            = FL-75-k115 Fuel Tank (N-Series)       // EA-F768 Fuel Tank
+		#LOC_NFLaunchVehicles_nflv-fueltank-75-5_title            = FL-75-k230 Fuel Tank (N-Series)       // EA-F1536 Fuel Tank
```

Comment:
Why you changed E-series to N-series for 7.5m?
It was like reference to the previous names: 5.0m - NR, 7.5m - EA

Reply:
found it - `N-Series -- Near Future Launch Vehicles`
better leave it as it was before (N - 5.0m, E - 7.5m) or make it more clear, like (NFLV)

Reply:
I've switched this back.